docs/layers/arc-layer-64.md

```diff
@@ -0,0 +1,9 @@
+# Arc Layer 64
+
+The Arc Layer takes in paired latitude and longitude coordinated points and
+render them as arcs that links the starting and ending points.
+
+    import {ArcLayer64} from 'deck.gl';
+
+Inherits from all [Base Layer](/docs/layers/base-layer.md) and
+[Arc Layer](/docs/layers/arc-layer.md) properties.
```

Comment:
This makes it look like this is a subclass of ArcLayer, which it currently is not. Instead of `inherits`, say that `this layer has the same properties as`